src/index.tsx

```diff
@@ -344,6 +348,7 @@ const QRCodeCanvas = React.forwardRef(function QRCodeCanvas(
           setIsImageLoaded(true);
         }}
         ref={_image}
+        crossOrigin={imageSettings?.crossOrigin ?? 'anonymous'}
```

Comment:
I don't _love_ that this is duplicating the work in `getImageSettings`. I'd need to see if we can pull the call to that out of our `useEffect` in here or maybe just pull the default setting out to ensure no potential drift.